Update color detection tutorial (#1865)
viamrobotics/docs: docs/tutorials/services/try-viam-color-detection.md

We recommend you verify the desired color detection value under actual lighting conditions.
To determine the color value from the actual cam component image, you can use a pixel color tool, like [Color Picker for Chrome](https://chrome.google.com/webstore/detail/color-picker-for-chrome/clldacgmdnnanihiibdgemajcfkmfhia).

> If the color is not reliably detected, drag the `Hue Tolerance` slider to the right or navigate to the `Raw JSON` tab to increase the `hue_tolerance_pct`.

Reviewer: ```suggestion
If the color is not reliably detected, drag the **Hue Tolerance** slider to the right or navigate to the `Raw JSON` tab to increase the `hue_tolerance_pct`.
```
Since in this case "Hue Tolerance" isn't a string that is part of the code or the JSON, we don't use monospace. If it was "hue_tolerance" you would, because that'd be the exact string and would be referring to the JSON instead of to the UI.

Viam [camera](/components/camera/) components can be [physical](/components/camera/webcam/) like the one already configured on the rover, or virtual.
A virtual camera transforms the output from a physical camera.

To view output from the color detector overlaid on images from a physical camera, configure a transform camera:

> 1. Navigate to the **Components** tab in the Viam app.

Reviewer: ```suggestion
1. Navigate to the **Config** tab in the Viam app and click the **Components** subtab.
```

Viam [camera](/components/camera/) components can be [physical](/components/camera/webcam/) like the one already configured on the rover, or virtual.
A virtual camera transforms the output from a physical camera.

> To view output from the color detector overlaid on images from a physical camera, configure a [transform camera](/components/camera/transform/).

Reviewer: ```suggestion
To view output from the color detector overlaid on images from a physical camera, configure a [transform camera](/components/camera/transform/):
```
I think this can be the introduction to the list of steps so the transform camera link doesn't have to appear twice

> Navigate to the **Components** tab in the Viam app and click the **Components** subtab.

Reviewer: ```suggestion
1. Navigate to the **Components** tab in the Viam app and click the **Components** subtab.
```

> To create a [transform camera](/components/camera/transform/):

Reviewer: ```suggestion
```
This part's kinda redundant

Reviewer: Oh I'm sorry my suggestion had "components" in it twice instead of "config" and then "components"! Sorry. Making new suggestion.

> 

Reviewer: ```suggestion
```

> 1. Click **Create Component** in the lower-left corner of the page.
> 2. Select `camera` as the type.
> 3. Select `transform` as the model.
> 4. Enter a name, for example `detectionCam`, and click **Create**.

Reviewer: ```suggestion
2. Click **Create Component** in the lower-left corner of the page.
3. Select `camera` as the type.
4. Select `transform` as the model.
5. Enter a name, for example `detectionCam`, and click **Create**.
```

> {{<imgproc src="/tutorials/try-viam-color-detection/create-transform-camera.png" resize="900x" declaredimensions=true alt="The Viam app showing the Create Component panel populated with a camera component. The name is detectionCam, the type is camera, and the model is transform.">}}

Reviewer: Per offline convo, I'd say shrink this to something smaller (exact number up to you).
Or actually you could just delete this whole image since with the UI change we moved to just delete most (or all) of these. I missed this part of this tutorial as I was making the other UI change PRs--thanks for catching here.
```suggestion
{{<imgproc src="/tutorials/try-viam-color-detection/create-transform-camera.png" resize="500x" declaredimensions=true alt="The Viam app showing the Create Component panel populated with a camera component. The name is detectionCam, the type is camera, and the model is transform.">}}
```

> Navigate to the **Components** tab in the Viam app and scroll to the **Create Component** section.
> To create a [transform camera](/components/camera/transform/):

Reviewer: ```suggestion
1. Navigate to the **Components** tab in the Viam app and click the **Components** subtab.
```
Also on line 126, change the period to a colon, and take out "you need to"

> 1. Click **Create Component**.

Reviewer: ```suggestion
1. Click **Create Component** in the lower-left corner of the page.
```

2. Select `camera` as the type.
3. Select `transform` as the model.

> 4. Enter a name for **Name**, for example `detectionCam`, and click **Create**.

Reviewer: ```suggestion
4. Enter a name, for example `detectionCam`, and click **Create**.
```

| `source` | `cam` | The name of the physical camera on the rover, which provides the visual feed to get detections from. |
| `attributes` | | The attributes of this detectionCam. |
| `attributes.detector_name` | `my_color_detector` | The name of this detectionCam. |
| `attributes.confidence_threshold` | `0.3` (30%) |  The percentage of confidence needed by the detection service to identify a color. |
| `type` | `detections` | The type of the component. |

> {{<imgproc src="/tutorials/try-viam-color-detection/transform-camera-attributes.png" resize="900x" declaredimensions=true alt="The Viam app showing the detectionCam component section. The Attributes section contains a skeleton configuration, including source, pipeline, type, and attributes. The Attributes Guide section lists the available camera component attributes. There are buttons labeled Data Capture Configuration, and Frame, and a drop-down labeled, Depends On. On the upper right there is a trash bin icon.">}}

Reviewer: Per offline convo, it makes sense for this one to use the markdown formatting `![...]` since it's a full width one.
